echopype/convert/set_groups_azfp.py

```diff
@@ -177,7 +179,9 @@ def set_platform(self) -> xr.Dataset:
         tilt_y = [np.nan] if np.isnan(unpacked_data["tilt_y"]).all() else unpacked_data["tilt_y"]
         if (len(tilt_x) == 1 and np.isnan(tilt_x)) and (len(tilt_y) == 1 and np.isnan(tilt_y)):
             time2 = [self.parser_obj.ping_time[0]]
+            depth = [depth[0]]
         else:
+            depth = -1 * depth
             time2 = self.parser_obj.ping_time
```

Comment:
Hmm, I'm just realizing that `pressure` may have its own time dimension that's different from `time2`! Let me think about this ...

Reply:
I also need to think about the `-1` factor, and our statement in the echopype documentation that `vertical_offset = -pressure_sensor_depth` ....

One thing we need to account for is whether the echosounder is pointing up or down. I need to think about that, too.

Reply:
> Hmm, I'm just realizing that `pressure` may have its own time dimension that's different from `time2`! Let me think about this ...

The code i just wrote is a work around since when both `tilt_x` and `tilt_y` are all `np.nan` then we set `time2` dimension as 1  but `pressure` data might be present when they `np.nan`.I think `pressure` might have it's own dimension?

Reply:
Dealing with the time dimensions here may be a bit tricky. Yes, `pressure` may have its own time dimension, but I'll need to ask about this.

Reply:
Updated!